src/app/account/additional-components/account-flows-tab/account-flows-tab.types.ts

```diff
@@ -391,4 +392,10 @@ export interface DashboardFiltersOptions {
     selectedFlowProcessStates: FlowProcessEffectiveState[];
     minConsecutiveFailures: number;
     isFirstInitialization: boolean;
+    applyFilters: boolean;
+}
+
+export interface ResultFlows {
+    totalCount: number;
+    nodes: AccountFlowProcessCard[];
 }
```

Comment:
The name is not so good. These are not flows, but cards. And it's more a listing, than a result. I would say `FlowProcessCardListing` or something like that.